packages/meteor-wallet/src/lib/meteor-wallet.ts

```diff
@@ -218,6 +219,12 @@ export function setupMeteorWallet({
   deprecated = false,
 }: MeteorWalletParams_Injected = {}): WalletModuleFactory<InjectedWallet> {
   return async () => {
+    const mobile = isMobile();
+
+    if (mobile) {
```

Comment:
This check is not needed for meteor wallet because it's not an extension wallet.